portfolio/src/main/webapp/script.js

```diff
@@ -31,4 +31,29 @@ function addRandomQuote() {
 
     console.log(quote);
 
-}
\ No newline at end of file
+}
+
+/** Fetches the hello world and adds it to the page. */
+async function getHelloWorld() {
+    const responseFromServer = await fetch('/hello');
+    const textFromResponse = await responseFromServer.text();
+  
+    const helloContainer = document.getElementById('hello-container');
+    helloContainer.innerText = textFromResponse;
+
+  }
+
+async function getShow(){
+     // Send a request to /hello.
+     const responseFromServer = await fetch('/show');
+     const shows = await responseFromServer.json();
+
+     //Pick a random show
```

Comment:
nit: leave a space after the "//" to begin the comment.

Reply:
Done